src/RunTime.cpp

```diff
@@ -0,0 +1,406 @@
+#ifdef __cplusplus
+#include "RunTime.h"
+#endif
+
+/**
+ * @brief Process the run time options
```

Comment:
Add how this can be called from Python. 
Is it the preferred way to do domain decomposition from Python now ?

Reply:
It is not designed for being called from Python, only for the shell. But Python can run shell commands using subprocess.call.